src/reducers/index.js

```diff
@@ -36,5 +40,7 @@ export default combineReducers({
   recipe,
   roles,
   toast,
-  users
+  users,
+  vendor,
+  vendors
```

Comment:
Is there a test that needs updating for this?

Reply:
Now that you mention it... it seems like there was a test for reducers' index, but it's not in the master branch.